tinyagentos/routes/video.py

```
@@ -179,13 +202,89 @@ async def generate_video(request: Request, body: VideoGenerateRequest):
+            status_code=503,
+        )
+
+    seed = body.seed if body.seed is not None else random.randint(0, 2**32 - 1)
+
+    store = await _get_video_job_store(request.app)
+    job_id = await store.create_job()
+
+    task_set = getattr(request.app.state, "_background_tasks", None)
+    coro = _run_generation_job(request.app, job_id, body, seed, backend_url)
+    if task_set is None:
+        asyncio.create_task(coro)
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)